platform/ethereum/collections_client.go

```diff
@@ -51,12 +51,15 @@ func (c CollectionsClient) GetCollectibles(owner string, collectibleID string) (
 		"owner": {owner},
 		"limit": {strconv.Itoa(300)},
 	}
+
 	for _, i := range collection.Contracts {
+		if _, ok := slugTokens[i.Type]; ok {
+			uriValues.Set("collection", collection.Slug)
```

Comment:
What if hasn't slugs?

Reply:
the Blockatlas going to search by contract address